linter/ginkgo_linter.go

```diff
@@ -41,10 +41,6 @@ func (l *GinkgoLinter) Run(pass *analysis.Pass) (any, error) {
 		gomegaHndlr := gomegahandler.GetGomegaHandler(file, pass)
 		ginkgoHndlr := ginkgohandler.GetGinkgoHandler(file)
 
-		if gomegaHndlr == nil && ginkgoHndlr == nil { // no gomega or ginkgo imports => no use in gomega in this file; nothing to do here
```

Comment:
This is pretty important check, made to improve performance. We don't won't to dig in files that do not include code that uses gomega and/or ginkgo (or wrappers). e.g. all the production code. For these files we want to fast return and exit.

Please restore this code.